app/renderer/components/preferences/payment/enabledContent.js

```diff
@@ -662,6 +675,23 @@ const styles = StyleSheet.create({
     fontSize: globalStyles.payments.fontSize.regular,
     lineHeight: 1.5,
     marginTop: globalStyles.spacing.panelPadding
+  },
+
+  fundsAmount__item: {
+    marginBottom: '4px',
+    width: '215px',
+    fontSize: '14.5px'
```

Comment:
Is there a reason this isn't just `15px`? it looked fine with it for me

Reply:
We use 14.5px in other places by default, so that's why I set it here as well